content/guides/farmlands-pos-integration.md

```diff
@@ -1,7 +1,10 @@
 ---
-title: Farmlands POS Integration Guide
+title: POS Integration Guide
```

Comment:
The title on this page (Farmlands POS Integration Guide) was different to the title on the nav (POS Integration Guide).

These two titles should match to avoid confusion for the user. Changing the title on the nav to "Farmlands POS Integration Guide" causes the title to wrap which doesn't look very nice.

Reply:
@Ray2126 we had a few instances like this so it may be best to go through them with Tom and check whether we want to rename anything else.
Another example is that `Merchant Integration Error Handling` was just `Error Handling` in the nav.

Reply:
Good catch!

<img width="576" alt="image" src="https://user-images.githubusercontent.com/50732795/215626197-6b3a05ca-aa35-4757-879e-52f9f0665b1d.png">

@tom-blake Above is a screenshot with the local nav on the left and the production nav on the right. I think we can rename them all so:
- QR Code Flow for Merchants --> QR Code Flow
- Barcode Flow for Merchants --> Barcode Flow
- Merchant Integration Error Handling --> Error Handling

This change will mean the titles on the respective pages will also change. I think this is okay because they all come under the `Merchant Integrations` section.

Reply:
The missing context will also be remedied once we add breadcrumbs.

Reply:
I've had a chat with Tom and we decided to put a Nav title in the front matter. This way, the nav and the page title is decoupled. The nav will default to the page title if the Nav title does not exist.